### DISCUSSION TITLE
A question on the periodic-hill case

### DISCUSSION THREAD

**Discussion by esaka-forever** [xcompact3d/Incompact3d#184] [Q&A] [answered]
Hi,
I have tried to run a DNS for periodic hills using this code. Unlike the other cases, there is no example code for the periodic hills for which IBM should be used. So I just copy the input.i3d from the cylinder example case (which I believe IBM is also used), and change the flow type to period hills.  After I run the code, the velocity diverges in the first step, regardless of whether I set iibm=1 or 2 (iibm is the Flag for immersed boundary method) or how I adjust the timesteps or Reynolds number. I highly appreciate it if anyone can offer any help or hint on what else I should change.

**Comment by slaizet** [comment, marked as answer]
Hi,
We will aim to provide an example code for the periodic hills. There are few changes to be made with respect to the cylinder case, for example the boundary conditions & the initial condition. I will create an issue so that we can start working on an example code for the periodic hills.
Best,
Sylvain

**Reply by esaka-forever** [reply]
Glad to know that. Thank you!
